code/modules/cargo/items/science.dm

```diff
@@ -45,7 +45,7 @@
 	name = "monkey cube box"
 	supplier = "nanotrasen"
 	description = "Drymate brand monkey cubes. Just add water!"
-	price = 60
+	price = 1600
```

Comment:
This seems a bit excessive - monkey cubes are more expensive than a canister of phoron?